app/views/users/_edit_primary.html.erb

```diff
@@ -7,25 +7,38 @@
       <%= f.file_field :avatar %>
       <%= f.hidden_field :avatar_cache %>
       <span class="help-block">JPG, GIF, or PNG (less than 2MB)</span>
+
```

Comment:
Nothing to do with Zotero other than this living in the UI right next to the Zotero widget, which is why I spruced it up in this PR.  Happy to remove if you think that's best.